spinta/dimensions/enum/commands.py

```diff
@@ -26,6 +27,18 @@ def check(
     ))
 
 
+@overload
+@commands.check.register(Context, EnumItem, DataType, Expr)
+def check(
+    context: Context,
+    item: EnumItem,
+    dtype: DataType,
+    value: Expr,
+):
+    output_value = value.args[-1] # TODO not sure if its good approach for the future expressions.
+    if output_value:
+        dtype.load(output_value)
```

Comment:
You're calling `load` from a `check` command is that right? Haven't seen an occurrence where it would not be the same command, but might be just missing context.

Reply:
This load behaves more like `validate` action. In case of a `swap(null, '-')` I check if second parameter ('-' in example) is correct type, same as destination (string)